### ISSUE TITLE
1.18.2 Missing Usine

### ISSUE THREAD

**Issue by Armokthespideroverlord** [jtorleonstudios/HUB-Awesome-Dungeon#7]
While playing around with this mod, I've noticed that one of its structures, the Usine, doesn't seem to generate at all, yet weirdly I've gotten the advancement for locating one when teleporting to one.

I'm using the following structure adding mods in addition to Awesome Dungeons on forge 40.1.69:
- Awesome Dungeons Ocean Edition
- Bygone Nether
- Dungeons Enhanced
- Habitat
- Infernal Expansion
- Quark
- Supplementaries
- Valhelsia Structures
- YUNG's Better Desert Temples
- YUNG's Better Dungeons
- YUNG's Better Mineshafts
- YUNG's Better Strongholds
- YUNG's Better Witch Huts
- YUNG's Extras

And while not structure mods per say, I'm using these mods which happen to alter vanilla dungeons or add some kind of structure-like world gen:
- Savage & Ravage
- Snowy Spirit
- Waystones

**Comment by BBoldt** [comment]
Hey, I've been doing a lot of customization of Awesome Dungeons using datapacks for my modpack, and I discovered a typo that is causing this problem.

in awesomedungeon\worldgen\template_pool\the_usine\start_pool.json

`{
	"name": "awesomedungeon:the_usine/start_pool",
	"fallback": "minecraft:empty",
	"elements": [
		{
			"weight": 1,
			"element": {
				"location": "awesomedungeon:the_usiner",
				"processors": "minecraft:empty",
				"projection": "rigid",
				"element_type": "minecraft:single_pool_element"
			}
		}
	]
}`
the line:

` "location": "awesomedungeon:the_usiner", `

should be:

` "location": "awesomedungeon:the_usine", `

**Comment by 48004800** [comment]
Version 1.19.4 also has this issue

**Comment by jtorleonstudios** [comment]
Yes we will reactivate the structure when it is reworked. because it is not playable in the current state (unwanted block)
